bin/tape

```diff
@@ -26,10 +26,13 @@ opts.require.forEach(function(module) {
 });
 
 opts._.forEach(function (arg) {
-    glob(arg, function (err, files) {
-        files.forEach(function (file) {
-            require(resolvePath(cwd, file));
-        });
+    var files;
+    try {
+        files = glob.sync(arg);
```

Comment:
To my reading, the original code would still throw the `undefined.forEach` error, it'd just be thrown from the async callback.